nexus/src/app/sagas/instance_create.rs

```diff
@@ -106,6 +106,9 @@ declare_saga_actions! {
     INSTANCE_ENSURE -> "instance_ensure" {
         + sic_instance_ensure
     }
+    V2P_ENSURE -> "v2p_ensure" {
```

Comment:
I can't leave comments on an arbitrary part of the file (ugh) so leaving this here because it's closely related: if this ordering changes and `V2P_ENSURE` gets an undo step, we should also update the tests in this module to check that no V2P mappings exist after a saga is torn down (there are a bunch of functions down there that check this sort of thing for other side effects).

Reply:
done in 48754ea2